main/main.cpp

```diff
@@ -1445,6 +1445,11 @@ bool Main::start() {
 			Viewport::Usage usage = Viewport::Usage(int(GLOBAL_GET("rendering/quality/intended_usage/framebuffer_allocation")));
 			sml->get_root()->set_usage(usage);
 
+			int msaa_mode = GLOBAL_GET("rendering/quality/filters/msaa");
```

Comment:
this code is already in SceneTree::initialize(), I wonder why it may not have any effect if this is the case

Reply:
I will check that.

Reply:
I think there was a mistake for testing msaa option.
it respects msaa well.